docs/ros/tutorials/_category_.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Are we adding a tutorial for ROS 1 Noetic to ROS 2 Humble upgrades? Refer to https://docs.clearpathrobotics.com/docs/robots/outdoor_robots/husky/tutorials_husky/tutorials_husky_ros1#backing-up-robot-configuration and https://clearpathrobotics.com/assets/guides/melodic/melodic-to-noetic/index.html.

For the Client Success team, this will be the single most important part of the Humble documentation.

Reply:
Added to #128